README.md

```diff
@@ -21,6 +21,7 @@ These bindings allow Azure Functions to integrate with [Azure SignalR Service](h
 - Use Azure Functions (any language supported by V2) to send messages to a single user, or all the users in a group.
 - Use Azure Functions (any language supported by V2) to manage group users like add/remove a single user in a group.
 - Example scenarios include: broadcast messages to a SignalR Service hub on HTTP requests and events from Cosmos DB change feed, Event Hub, Event Grid, etc
+- Set up multiple Azure SignalR Service instances for Resiliency and disaster recovery. See [Multiple SignalR service endpoint support](./docs/sharding.md).
```

Comment:
For setting up xxx, see xxx

Reply:
Here's a list of supported scenarios, suggest state the scenario instead of guide something. How about: 
> Use multiple Azure SignalR Service instances for resiliency and disaster recovery in Azure Functions. See details in xxx.

---
In reply to: [592242426](https://github.com/Azure/azure-functions-signalrservice-extension/pull/179#discussion_r592242426) [](ancestors = 592242426)